packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -123,7 +141,10 @@ export function cloudflare<
 			const miniflare = new Miniflare({
 				workers: workers.map((workerOptions) => {
 					const wrappers = [
-						`import { createWorkerEntrypointWrapper } from '${runnerPath}';`,
+						// Note: this import relies on the `relativeRunnerPathSegments` array because using the full `runnerPath`
+						//       in windows would not work since `runnerPath` would start with `Z:` and workerd would
+						//       not know how to properly handle such path
+						`import { createWorkerEntrypointWrapper } from '${['.', ...relativeRunnerPathSegments].join('/')}';`,
```

Comment:
Sorry to be picky but can we put this path in a variable with the others. It doesn't feel right for the logic to live here.

Reply:
Actually, don't worry about this now. It can be refactored later.

Reply:
lol sorry I already did 😅